.gitignore

```diff
@@ -0,0 +1,23 @@
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
```

Comment:
Why would archives like these ever show up here?

Reply:
This may be better removed. I just copied the standard .gitignore that github has for each language. As I am not a Java developer and don't know the files the Java tools create. I did not judge. It was a little annoying to have the .class files show up after build the project.